src/Main/CompilerNotifier.cpp

```diff
@@ -42,7 +42,7 @@ void CompilerNotifier::notifyStart() {
 
 void CompilerNotifier::notifyFinish() {
   if (m_console) {
-    m_queue.dequeue();
+    if (!m_queue.isEmpty()) m_queue.dequeue();
```

Comment:
I haven't found the test when I got crash on this. Maybe it was my local changes. 
But any way this **if**  should be here since according to the documentation QQueue::dequeue assumes that container isn't empty.